src/assets/styles/base.scss

```diff
@@ -64,21 +48,30 @@ h1 {
   font-size: 2rem;
 }
 
+h4 {
+  letter-spacing: 0;
+  font-size: 0.9375rem; // 15px
+  font-weight: 700; 
+}
+
 p {
   font-weight: 400;
+  color: $gray7;
 }
 
-.app-action {
-  color: $app-blue !important;
+.font-13 {
+  font-size: 0.8125rem; // 13px
```

Comment:
Unique sizing!

Reply:
This was existing actually - used by View and Change Company Information button and Dissolve This Company button. I just made it more explicit and global.

Reply:
Guess how many font sizes we use in this app... (rhetorical question)

(Too many. I think Tracey wants us to look at this some time in the future.)

Reply:
haha yes, i can't imagine :)